src/components/MessageComposer/composer-crash.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ComposerCrash = ({containerClasses, resetError}) => (
+  <div className={containerClasses}>
+    <div className="error-title">Oops, the message composer broke</div>
+    <div className="error-message">Please try clicking reload below or refresh the app</div>
```

Comment:
these should be translated text. i think this should be handled in our app so we have control over what error should be displayed

Reply:
I just put it here to keep it simple since *hopefully* users never see this. Should I pass just the strings from the client to display or the entire component? It also makes it easier to use intl since the composer repo doesn't have intl